config/excel.php

```diff
@@ -259,6 +259,20 @@
         'illuminate' => [
             'store' => null,
         ],
+
+        /*
+        |--------------------------------------------------------------------------
+        | Cache Time-to-live (TTL)
+        |--------------------------------------------------------------------------
+        |
+        | The TTL of items written to cache. If you want to keep the items cached
+        | indefinitely, set this to null.  Otherwise, set a number of seconds,
+        | a \DateInterval, or a callable.
+        |
+        | Allowable types: callable|\DateInterval|int|null
+        |
+         */
+        'default_ttl' => null,
```

Comment:
I think we can set a sensible default here, I don't think anyone is expecting it to be cached forever.

Reply:
I set it to 3 hours. 👍